kubetest2-gke/deployer/build/gke_make.go

```diff
@@ -112,6 +116,23 @@ func (gmb *GKEMake) Stage(version string) error {
 	if err := gmb.runWithActions(os.Stdout, os.Stderr, []gkeBuildAction{stage}, args...); err != nil {
 		return err
 	}
+
+	if gmb.UpdateLatest {
+		r := regexp.MustCompile(gkeMinorVersionRegex)
+		m := r.FindStringSubmatch(version)
+		if len(m) < 2 {
+			return fmt.Errorf("failed to update the latest version because can't find the minor version of %s", version)
```

Comment:
let's make this a warning
and update just `latest.txt` in this case.

Reply:
Done